org.adempiere.base/src/org/compiere/print/DataEngine.java

```diff
@@ -1185,15 +1195,25 @@ else if (pdc.getDisplayType() == DisplayType.DateTime)
 									valueString = DisplayType.getDateFormat(pdc.getDisplayType(), m_language).format(value);
 								if (format.getTableFormat().isPrintFunctionSymbols())		//	Translate Sum, etc.
 									valueString	+= PrintDataFunction.getFunctionSymbol(functions[f]);
+								if (m_group.isFunctionColumn(pdc.getAD_PrintFormatItem_ID(), functions[f])) {
+									Serializable groupValue = m_group.getValue(pdc.getAD_PrintFormatItem_ID(), 
+											pdc.getAD_PrintFormatItem_ID(), functions[f]);
+									if(groupValue instanceof Timestamp)
+										groupValue = DisplayType.getDateFormat(pdc.getDisplayType()).format((Timestamp)groupValue);
+									if(groupValue != null)
+										valueString += ": " + groupValue;
+								}
```

Comment:
same as my question above, why the need of this after the timestamp check above.